### ISSUE TITLE
Article never explains what exactly a Store Candidate is

### ISSUE THREAD

**Issue by yashsway** [AdobeDocs/experience-manager-65.en#52]
Issue in help/sites-developing/ch-extend.md

- How is this related to the ContextHub store?
- Why is it needed?
- What does it do?

**Comment by bohnertchris** [comment]
Hello @yashsway 

ContextHub stores are created from registered store candidates. To create a custom store, you need to create and register a store candidate. You can have few implementations of one store, and during initialization of contexthub, all candidates where the `applies` function returns true are considered, but the one with highest priority is used.

For example for localization store, you can check if customer have google maps api key or not. Depending on that, simpler or more advanced functionality can be loaded. In the `applies` function of the store implementation you might check if a Google Maps API key is present or not.

In short, in the ContextHub you mention which stores you want to load, but in some cases you want to create few implementations that might apply to some cases and not, candidates allows you to choose the right implementation.

See the API documentation for more details on the `registerStoreCandidate(store, storeType, priority, applies)` and `applies` in particular.

https://docs.adobe.com/content/help/en/experience-manager-65/developing/personlization/contexthub-api.html#contexthub-utils-storecandidates

**Comment by bohnertchris** [comment]
@yashsway was my explanation clear? Do you have any questions about the documentation?

**Comment by bohnertchris** [comment]
Closing this issue. If you find any issues with the documentation please open a new issue.
